src/vl-tooltip.js

```diff
@@ -253,6 +251,14 @@
         default:
       }
     }
+    
+    // supplement bin from fieldDef, user should never have to provide bin
+    if (fieldDef.bin) {
+      supplementedFieldOption.bin = true;
+      supplementedFieldOption.formatType = 'string'; // we show bin range as string (e.g. "5-10")
```

Comment:
Ok, then I guess I can remove `string` as an option for `formatType`. It makes sense as strings don't have format specifier anyways. I guess I will do this in a different PR?